pkgs/development/python-modules/pocketsphinx/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  cmake,
+  cython,
+  fetchFromGitHub,
+  memory-profiler,
+  ninja,
+  pathspec,
+  pocketsphinx,
+  pytestCheckHook,
+  scikit-build,
+  scikit-build-core,
+  sounddevice,
+}:
+
+buildPythonPackage rec {
+  inherit (pocketsphinx) version src;
+  pname = "pocketsphinx";
+  pyproject = true;
+
+  dontUseCmakeConfigure = true;
+
+  env.CMAKE_ARGS = lib.cmakeBool "USE_INSTALLED_POCKETSPHINX" true;
+
+  buildInputs = [ pocketsphinx ];
+
+  build-system = [
+    cmake
+    cython
+    ninja
+    pathspec
+    scikit-build
```

Comment:
```suggestion
```
I don't think we need `scikit-build` because we have `scikit-build-core`.

Reply:
Yep, not needed